<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_form</class>
 <widget class="QWidget" name="settings_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Expression Recognizer</string>
  </property>
  <widget class="QFrame" name="color_F">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>600</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="color_L">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;COLOR(RGB):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="R_SB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>50</horstretch>
        <verstretch>50</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="G_SB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>50</horstretch>
        <verstretch>50</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="B_SB">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="size_F">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>390</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="size_L">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;SIZE:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="W_SB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>50</horstretch>
        <verstretch>30</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>120</number>
      </property>
      <property name="maximum">
       <number>800</number>
      </property>
      <property name="value">
       <number>640</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="H_SB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>50</horstretch>
        <verstretch>30</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>80</number>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
      <property name="value">
       <number>480</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="back_BTN">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
